### ISSUE TITLE
Add Ts_subseq_isconstant to gpu_ostinato

### ISSUE THREAD

**Issue by seanlaw** [TDAmeritrade/stumpy#880]
Currently the `Ts_subseq_isconstant` parameter is missing from `gpu_ostinato` and needs to be added

**Comment by seanlaw** [comment]
@NimaSarajpoor Are you able to handle this? Thanks in advance

**Comment by NimaSarajpoor** [comment]
@seanlaw 
I think you have already handled it. Right? (see https://github.com/TDAmeritrade/stumpy/commit/c7967d0df99bb24036d5b03120d6e24080ebf5b3)

**Comment by seanlaw** [comment]
No, I have only updated the docstring but they are missing `isconstant` support in the actual function itself
